Formula/latexml.rb

```diff
@@ -13,6 +13,10 @@ class Latexml < Formula
     sha256 "b911ac9897012edcc7c32d96785e4ca3830ce8cbddff78da0942263c7fb0d0bb" => :sierra
   end
 
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "perl"
+  
```

Comment:
`brew style` is picking up on trailing whitespace on this line

Reply:
Thanks. And it turns out some `—online` audits are failing. 😳

Reply:
Fixed style and audits that I could reproduce the failures for.